docs/shared/using-nx/affected.md

```diff
@@ -244,6 +244,22 @@ Nx provides two methods to exclude glob patterns (files and folders) from `affec
 - Glob patterns defined in your `.gitignore` file are ignored.
 - Glob patterns defined in an optional `.nxignore` file are ignored.
 
+## Marking Projects Affected by Dependency Updates
+
+By default Nx will mark all projects as affected whenever your package manager's lock file changes. This is meant to be a failsafe in case Nx misses a project that should be affected by a dependency update. If you'd like to opt in to a smarter behavior you can configure Nx to only mark projects as affected if they actually depend on the updated packages.
```

Comment:
- Missing comma after "default - `By default, Nx...`
- I would be more precise on "this is" for documentation, and we can remove `mean to be` since it's superfluous -- `This behavior is a failsafe in case Nx...`